regression-test/suites/query_p0/system/test_table_options.groovy

```diff
@@ -185,5 +178,32 @@ suite("test_table_options") {
     """
 
     qt_select """select * from information_schema.table_options where table_schema=\"${dbName}\" order by TABLE_NAME; """
-    sql "drop database if exists ${dbName}"
+    sql """
+        drop table test_row_column_page_size2;
+    """    
+    qt_select """select * from information_schema.table_options where table_schema=\"${dbName}\" order by TABLE_NAME; """    
+
+    def user = "table_options_user"
+    sql "DROP USER IF EXISTS ${user}"
+    sql "CREATE USER ${user} IDENTIFIED BY '123abc!@#'"
+    sql "GRANT SELECT_PRIV ON information_schema.table_properties  TO ${user}"
+
+    def tokens = context.config.jdbcUrl.split('/')
+    def url=tokens[0] + "//" + tokens[2] + "/" + "information_schema" + "?"
+
+    connect(user=user, password='123abc!@#', url=url) {
+       qt_select """select * from information_schema.table_options ORDER BY TABLE_NAME ASC; """
```

Comment:
For easy for reviwer to check every qt's output, better to change as follow:
1 rename ```qt_select``` to ```qt_select_check_1```,```qt_select_check_2``` and ```qt_select_check_3``` .
2 gant one table's priv intead of DB priv, so the output is less and can be reviewed manually.

Reply:
thanks, I have handled it.